application/Hand.py

```diff
@@ -1,4 +1,30 @@
-class Hand: 
-  def __init__(self, cards, startingPlayer):
-    hand = cards
-    player = startingPlayer
+import psycopg2
+import random
+import application.Card as Card
+
+# Probably better to get rid of the hand class once there is a better place for createHands()
+class Hand:
```

Comment:
Agreed. I wrote this before the first mention of SQL Alchemy. We're going to have to consider where DB connections will be made if more state information is going to be kept on the DB. It would be nice to keep all the DB stuff in one file, for maintainability's sake.